lib/css/components/_stacks-links.less

```diff
@@ -94,6 +94,25 @@ a,
     &.s-link__underlined {
         text-decoration: underline !important;
     }
+
+    &.s-link__dropdown {
+        position: relative;
+        padding-right: 0.9em;
+
+        &:after {
+            content: "";
+            position: absolute;
+            z-index: @zi-active;
+            top: calc(50% - 2px);
+            right: 0;
+            border-style: solid;
+            border-width: @su4;
+            border-top-width: @su4;
+            border-bottom-width: 0;
+            border-color: currentColor transparent;
+            pointer-events: none;
+        }
```

Comment:
### Nitpick Supreme
Absolutely positioned, statically sized stuff in elements who's parents could be of different (font) sizes or could have their relative positioning overridden makes me a little itchy.

I know this is also how it's handled on buttons (and I've either expressed this same discomfort there or, at least, thought it loudly). I think this could be safely handled with relative positioning and a few _em_ values to have it work in all sorts of weird contexts where these classes have been remixed in ways we might not anticipate, but this is _real_ low on the list of stuff to deal with. Ignore me!